src/SmartComponents/Recs/List.js

```diff
@@ -56,6 +57,12 @@ const List = () => {
     history.push(tab === 1 ? '/recommendations/pathways' : '/recommendations');
   };
 
+  const { flagsReady } = useFlagsStatus();
+
+  const isAppTourEnabled =
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    (flagsReady && useFlag(featureFlags.appTour)) || false;
```

Comment:
tse, tse, tse. :D Why not pack the `useFlagStatus` and `useFlag` into a hook like `useFeatureFlag('appTour')` and use `useState` for the flagReady?

Reply:
Also, is there a benefit to using && and || instead of using ternary operator (? and :)? You could get rid of some parentheses.

Reply:
@bastilian wondering how I can avoid conditional useFlag hook call by using suggested hook?

Reply:
I need to call only if flags are ready.

Reply:
@leSamo I am too much used to logical operators. I will update this to use ternary operators.

Reply:
Never mind, looks like useFlag hook handles non-existing flags nicely. It returns false. It means even if the flags are not fetched from backend, it will not break the page.